Source/DafnyCore/AST/Node.cs

```diff
@@ -134,6 +77,78 @@ public abstract class Node : INode {
     }
     set => rangeToken = value;
   }
+}
+
+public abstract class RangeNode : Node {
+  public override RangeToken RangeToken { get; set; } // TODO remove set when TokenNode is gone.
```

Comment:
```suggestion
  public override RangeToken RangeToken { get; set; } // TODO remove set when TokenNode is gone.
```
Will TokenNode ever go away?

Reply:
Jep. Next PRs ^,^